src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ScopeTab.tsx

```diff
@@ -3,36 +3,33 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Project } from 'shared';
-import { Box, Typography } from '@mui/material';
-
-const styles = {
-  bulletList: {
-    paddingLeft: '35px',
-    marginBottom: '0em'
-  }
-};
+import { DescriptionBullet, Project } from 'shared';
+import { Typography } from '@mui/material';
 
 export const ScopeTab = ({ project }: { project: Project }) => {
-  const descriptionBullets = project.descriptionBullets.map((bullet, index) => <li key={index}>{bullet}</li>);
+  const descriptoinBulletsSplitByType = new Map<string, DescriptionBullet[]>();
+  for (const bullet of project.descriptionBullets) {
+    if (bullet.dateDeleted) continue;
+    if (!descriptoinBulletsSplitByType.has(bullet.type)) {
+      descriptoinBulletsSplitByType.set(bullet.type, []);
+    }
+    descriptoinBulletsSplitByType.get(bullet.type)!.push(bullet);
+  }
 
   return (
-    <Box sx={{ display: 'flex', flexDirection: 'row', mt: 2 }}>
-      <Box width="50%">
-        <Box>
-          <Box sx={{ display: 'flex', justifyContent: 'space-between', mb: 1 }}>
-            <Typography
-              variant="h5"
-              sx={{
-                cursor: 'pointer'
-              }}
-            >
-              Description Bullets
-            </Typography>
-          </Box>
-          <ul style={styles.bulletList}>{descriptionBullets}</ul>
-        </Box>
-      </Box>
-    </Box>
+    <>
+      {Array.from(descriptoinBulletsSplitByType.entries()).map(([type, bullets]) => (
+        <>
+          <Typography key={type} variant="h6" gutterBottom>
+            {type}
+          </Typography>
+          <ul key={type}>
+            {bullets.map((db) => (
+              <li key={db.id}>{db.detail}</li>
+            ))}
+          </ul>
+        </>
+      ))}
```

Comment:
can't really test this because my local finishline is being weird and not loading rn, but I'm sure it's good